.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Hi, i think that we need to exclude .idea files from package.

Reply:
These files are actual only while developing package and have no sence for ready one

Reply:
Thanks for the response. I've removed the .idea files.

Reply:
Hi @timneil ,
Just figured out that you have updated package-lock.json with private npm repo links, and this breaks build. 
Could you please update lock file?
Best Regards,
Artem